README.md

```diff
@@ -1,4 +1,6 @@
-# Cryptographic hashing functions for Dart
+## Cryptographic hashing functions for Dart
+
+[![Build Status](https://travis-ci.org/dart-lang/crypto.svg?branch=master)](https://travis-ci.org/dart-lang/crypto)
 
 A set of cryptographic hashing functions implemented in pure Dart
```

Comment:
The list of hash algorithms should link to their implementations' dartdoc pages.

Reply:
ack